components/modal/Register/register.component.js

```diff
@@ -1,214 +1,216 @@
-import React, { Component } from 'react';
-import { Form, Input, Label, FormGroup, FormFeedback } from 'reactstrap';
-import { isEmail, isDecimal } from 'validator';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { isEmail } from 'validator';
+import PropTypes from 'prop-types';
 import classNames from './register.component.module.scss';
```

Comment:
`import classNames from './register.component.module.scss';`
Can be improved to 
`import styles from './register.component.module.scss';`

What are your thoughts?

Reply:
Considering the consistency in other files, I went with `classNames`.  My suggestion

Reply:
Ok